misc/python/materialize/parallel_workload/action.py

```diff
@@ -1130,6 +1130,7 @@ def __init__(
             "enable_compute_replica_expiration",
             "compute_replica_expiration_offset",
             "enable_compute_render_fueled_as_specific_collection",
+            "enable_compute_temporal_bucketing",
```

Comment:
What happens if we change the setting during a test run, could that be interesting to test?

Reply:
It only applies when rendering a dataflow. Would it be interesting to toggle it in this case?

Reply:
I think so